.circleci/config.base.yml

```diff
@@ -462,6 +462,8 @@ jobs:
         type: executor
         default: l_large
     executor: << parameters.os >>
+    environment:
+      AMPLIFY_PATH: /home/circleci/.npm-global/lib/node_modules/@aws-amplify/cli/bin/amplify
```

Comment:
this migration tests was missing this, all other migration tests were already specifying this.
There was a 'hack' in the split-e2e-tests script that allowed v5 migration tests to work without this line, but now thats been removed.

Reply:
is `/home/circleci/.amplify/bin/amplify` different from `/home/circleci/.npm-global/lib/node_modules/@aws-amplify/cli/bin/amplify` change set wise?

Reply:
the split-e2e-tests script was inserting this line for all migration tests;
This resolves to the binary that is installed via NPM prior to packaged CLI (v6 and earlier), which was at the time installed to .npm-globa/.../bin

Reply:
gotcha, so if I run `/home/circleci/.npm-global/lib/node_modules/@aws-amplify/cli/bin/amplify version` on that circle ci VM this will print v5 or v6 right ?

Reply:
yeah thats right